src/Serializer/SerializerContextBuilder.php

```diff
@@ -96,18 +100,6 @@ public function createFromRequest(Request $request, bool $normalization, array $
 
         unset($context[DocumentationNormalizer::SWAGGER_DEFINITION_NAME]);
 
-        if (isset($context['skip_null_values'])) {
-            return $context;
-        }
-
-        foreach ($resourceMetadata->getItemOperations() as $operation) {
-            if ('PATCH' === ($operation['method'] ?? '') && \in_array('application/merge-patch+json', $operation['input_formats']['json'] ?? [], true)) {
-                $context['skip_null_values'] = true;
-
-                break;
-            }
-        }
```

Comment:
this code clearly made a wrong assumption: if a resource supports PATCH I'll skip null values from the response.

Reply:
Did you see https://github.com/api-platform/core/issues/3159?